config.go

```diff
@@ -29,6 +29,9 @@ type lndConfig struct {
 	// MacaroonName is the name of the macaroon in macaroon dir to use.
 	MacaroonName string `long:"macaroonname" description:"The name of our macaroon in macaroon dir to use."`
 
+	// RPCTimeout is the timeout for rpc calls to lnd in seconds.
+	RPCTimeout int `long:"rpctimeout" description:"The timeout for rpc calls to lnd in seconds"`
```

Comment:
nit: this can be `time.Duration` directly instead of `int`, then the default can be `30 * time.Second`.

Reply:
Thank you so much! 

If I change this to `time.Duration` that would require the user to specify a unit like: `60s`. Is that OK? Or is it better to abstract this from the user so they only have to enter an int?

Reply:
Yes, I think that's okay. Perhaps we should add a hint to the description, in `lnd` we use a text like this: `Valid time units are {s, m, h}.`

Reply:
Cool, that makes sense. Thanks for the reference to `lnd`. I've pushed the update.

Reply:
Thanks! Just need to run `make fmt`, then this looks good.

Reply:
done